lib/state.js

```diff
@@ -81,13 +85,17 @@ function StateResolver(execState, expressions, config) {
   this.evaluatedExpressions_ = [];
   this.totalSize_ = 0;
 
-  // The 0-th object in the variable table is a sentinel 'buffer full' error
-  // message value
-  this.rawVariableTable_ = [ null ];
-  this.resolvedVariableTable_ = [
+  this.rawVariableTable_ = [ null, null, null ];
+  this.resolvedVariableTable_ = [];
+  this.resolvedVariableTable_[BUFFER_FULL_MESSAGE_INDEX] =
+    { status: new StatusMessage(StatusMessage.VARIABLE_VALUE,
+                                'Buffer full', true) };
+  this.resolvedVariableTable_[NATIVE_PROPERTY_MESSAGE_INDEX] =
+    { status: new StatusMessage(StatusMessage.VARIABLE_VALUE,
+                                'Native property', true) };
```

Comment:
I wonder if we should make this more descriptive. Something like 'Native properties are not available'?